task160_FinalTask/src/test/java/WebDriverSingleton/WebDriverSingleton.java

```diff
@@ -0,0 +1,37 @@
+package WebDriverSingleton;
+
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class WebDriverSingleton {
+    private static WebDriverSingleton object = null;
+    private WebDriver driver;
+
+
+    private WebDriverSingleton() {
+        //driver = new FirefoxDriver();
+        driver = new ChromeDriver();
```

Comment:
It will be nice to have parameter like "browser" and handle it here. For example, when I want to run tests in chrome: mvn clean test -Dbrowser=chrome

Reply:
I've added .properites file to my project. And implement a switch in Singleton which looks on the property value